ci.gradle

```diff
@@ -59,7 +59,7 @@ gitPublishCopy.dependsOn(scaladoc)
 task ci {
     description = "The main target for CI builds."
     group = "CI"
-    if (TravisEnvironment.isMasterBuild()) {
+     if (TravisEnvironment.isMasterBuild()) {
```

Comment:
🤔 why?

Reply:
btw. we're starting to be pretty tight into Travis here :) just saying...

Reply:
It was commented and I probably forgot the whitespace.